frontend/projects/ui/src/app/pages/marketplace-routes/marketplace-show/marketplace-show.page.ts

```diff
@@ -128,11 +128,15 @@ export class MarketplaceShowPage {
     await alert.present()
   }
 
-  async presentModalMd(title: string) {
+  async presentModalMd(title: 'license' | 'instructions') {
+    const contentPromise = this.marketplaceService.getPackageMarkdown(
```

Comment:
the license wont necessarily be markdown. it can be a regular text file. not sure if that matters.

Reply:
wont matter